index.js

```diff
@@ -142,17 +147,18 @@ module.exports = {
         }
 
         if (autoCaptureOptions.indexOf('exception') !== -1) {
-
-            nemo.driver.flow_.on('uncaughtException', function (exception) {
+            flow.once('uncaughtException', function (exception) {
+                if (!driver.session_) {
+                    return;
+                }
                 var filename = 'ScreenShot_onException-' + process.pid + '-' + new Date().getTime();
                 var screenShotFileName = path.resolve(screenShotPath, filename);
-                nemo.screenshot.snap(screenShotFileName).then(function () {
-                });
-                throw exception;
-            });
+                flow.await(nemo.screenshot.snap(screenShotFileName).then(function () {
```

Comment:
Oops. Sorry I got those switched while looking at the docs. I thought "wait" was the deprecated one.